test/quantization/test_auto_tracing.py

```diff
@@ -0,0 +1,924 @@
+                x = self.conv2(x)
+                return x
+
+        model_fp32 = M().eval()
+        qconfig = torch.quantization.default_qconfig
+        self._test_auto_tracing(model_fp32, qconfig, (torch.randn(1, 1, 2, 2),))
+
+    @unittest.skip('this depends on unsupported syntax detection, currently disabled')
+    def test_vovnet_sequential(self):
+
+        class SequentialAppendList(nn.Sequential):
+            def __init__(self, *args):
+                super(SequentialAppendList, self).__init__(*args)
+
+            def forward(self, x: torch.Tensor) -> torch.Tensor:
+                concat_list = []
+                for i, module in enumerate(self):
+                    if i == 0:
+                        concat_list.append(module(x))
+                    else:
+                        concat_list.append(module(concat_list[-1]))
+                x = torch.cat(concat_list, dim=1)
+                return x
+
+        m = SequentialAppendList(torch.nn.Conv2d(1, 1, 1)).eval()
+        qconfig = torch.quantization.default_qconfig
+        self._test_auto_tracing(m, qconfig, (torch.randn(1, 1, 1, 1),))
+
+    def test_unsupported_ops(self):
+        class M(torch.nn.Module):
+            def forward(self, x):
+                x = F.tanhshrink(x)
+                x = x + x
+                x = F.tanhshrink(x)
+                return x
+
+        model_fp32 = M().eval()
+        qconfig = torch.quantization.default_qconfig
+        self._test_auto_tracing(model_fp32, qconfig, (torch.randn(1, 1, 2, 2),))
+
+    def test_unknown_op_after_quantized(self):
+        class M(torch.nn.Module):
+            def forward(self, x):
+                x = x + x
+                std = x.std()
+                return std
+
+        model_fp32 = M().eval()
+        qconfig = torch.quantization.default_qconfig
+        self._test_auto_tracing(
+            model_fp32, qconfig, (torch.randn(1, 1, 2, 2),),
+            fuse_modules=False)
+
+    def test_embedding(self):
+        # test subclass
+
+        class EmbeddingSubclass(nn.Embedding):
+            pass
+
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.embedding = EmbeddingSubclass(1, 1)
+
+            def forward(self, x):
+                x = self.embedding(x)
+                return x
+
+        model_fp32 = M().eval()
+        qconfig = torch.quantization.default_dynamic_qconfig
+        self._test_auto_tracing(
+            model_fp32, qconfig, (torch.LongTensor([[0]]),),
+            fuse_modules=False)
+
+        # test regular embedding
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.embedding = nn.Embedding(1, 1)
+
+            def forward(self, x):
+                x = self.embedding(x)
+                return x
+
+        model_fp32 = M().eval()
+        qconfig = torch.quantization.default_dynamic_qconfig
```

Comment:
embeddings should require a float qparams qconfig `float_qparams_weight_only_qconfig`. Is this test ensuring that the embeddings get quantized?

Reply:
For now, this is just testing that models with embeddings can go through the APIs without crashing (embeddings do not get quantized).  Will add a comment to clarify.  There is already a TODO in `_quantize_dynamic_tracing.py` to fix this.